### ISSUE TITLE
Updating CarouselView.FormsPlugin to stable version 6.0.0 generates errors in Carousel renderers

### ISSUE THREAD

**Issue by NicolasKlaas** [UXDivers/Grial-UI-Kit-Support#775]
### Description
Just now I updated the` CarouselView.FormsPlugin` from version **5.2.0** to **6.0.0**. In this updated release the support for `AndroidX ` has been added, however this also makes all the **Carousel Renderers** outdated and returning errors. 
I am not sure the intial issues are fixed in this new version, can I safely delete all the carousel renderers which state _**'Once a new version of the carousel is published this file can be removed.'**_?

The `ExtendedCarouselViewRenderer.cs` does not state it can be removed, though it also returns errors. Can this also be safely removed?

Can the Grial package be updated with this new release of the CarouselView.FormsPlugin, and can the unnecesary renderers be removed (if applicable).

### Basic Information

- UXDivers.Grial 3.1.76 / CarouselView.FormsPlugin 6.0.0 
-  CarouselView.FormsPlugin 5.2.0 
- Platform Target Frameworks: 
  - Android: 10.0 

### Screenshots
![Capture](https://user-images.githubusercontent.com/34881984/106262906-f3e23e00-6223-11eb-9061-2aa7417b9a58.PNG)

**Comment by dirivero** [comment]
We added support for AndroidX in Grial version 3.1.76 so we move to using Xamarin.Forms CarouselView instead of CarouselView.FormsPlugin as version 6.0.0 was not published yet:
http://docs.grialkit.com/release-notes.html#release-notes-grial-3176

If you need the animations provided by the `ExtenedCarouselView` I recommend you to download the latest version of Grial and update your project with these:
 - get the new ExtendedCarouselView code to include in the .net standard project
 - get the new ExtendedCarouselViewRenderer for the Android project
 - remove android's vertical_viewpager.xml and horizontal_viewpager.xml files

Hope that helps.

**Comment by NicolasKlaas** [comment]
Ah no clue the Carousel view was also updated, I will read up on updates a bit better next time! Thank you anyway!

**Comment by turgaygncr** [comment]
Hi @dirivero 
I also have same errors on CarouselViewRenderer.cs and CirclePageIndicator.cs files on Android project.
There is no same page in the .net standart project on Grial. So i could not change these files.

Basic Information
UXDivers.Grial 3.2.82 / CarouselView.FormsPlugin 6.0.0
Xamarin.Forms 5.0.0.2012

Platform Target Frameworks:
Android: 10.0

![Ekran Resmi 2021-02-25 16 18 57](https://user-images.githubusercontent.com/5506941/109159160-5e37c100-7785-11eb-9c6f-db49944b59c9.png)
![Ekran Resmi 2021-02-25 16 19 16](https://user-images.githubusercontent.com/5506941/109159179-63950b80-7785-11eb-9969-12b8a7ac31f0.png)

**Comment by dirivero** [comment]
Hi @turgaygncr

You should remove those 2 files, I forgot to mentioned them in the previous answer.

**Comment by turgaygncr** [comment]
Hi @dirivero ,

I removed 2 files from android project. 
And also i removed ExtendedCarouselViewControl.cs in MainProject / Controls

After that i got errors bellow.

Thanks.

![Ekran Resmi 2021-02-25 16 44 41](https://user-images.githubusercontent.com/5506941/109162231-069b5480-7789-11eb-85f9-442e3e4039f1.png)
![Ekran Resmi 2021-02-25 16 44 49](https://user-images.githubusercontent.com/5506941/109162247-0ac77200-7789-11eb-8f6c-0ace034e5147.png)

**Comment by dirivero** [comment]
That error means that the `VideoCarouselHighlightsPage` uses the `ExtendedCarouselViewControl` in the XAML code. 

Did you get the new extended carousel code from Grial 3.2.82? Please also note that the new carousel is `ExtendedCarouselView` (without the ending `Control`), just rename it in that XAML page.

**Comment by turgaygncr** [comment]
Hi @dirivero 

Yes you are right, Thanks for all.
I changed all xaml files. The application has been compiled successfully. While running the Android project I got the following error.

I think the  "android.support.v4.content.FileProvider" provider in AndroidManifest.xml file gives error after Xamarin.forms  update.

![Ekran Resmi 2021-02-25 17 39 21](https://user-images.githubusercontent.com/5506941/109169458-e2dc0c80-7790-11eb-87c2-6a3a1e50f8b7.png)
